tests/network-tests/src/flows/content/createApp.ts

```diff
@@ -47,7 +49,7 @@ export async function createApp({ api, query }: FlowProps): Promise<void> {
     platforms: ['web', 'mobile'],
   }
 
-  await api.createApp(member.memberId, appOwnedByLead, appOwnedByLeadMetadata, true)
+  await api.createApp(leadId, appOwnedByLead, appOwnedByLeadMetadata, true)
```

Comment:
I think it's better to just make `memberId` arg optional (which will also imply whether the app is member-owned or lead-owned).
`leadId` is actually not ever a member id, it's a worker id